src/qibocal/protocols/characterization/two_qubit_interaction/cz_virtualz.py

```diff
@@ -395,24 +429,25 @@ def _plot(data: CZVirtualZData, fit: CZVirtualZResults, target: QubitPairId):
     fig1.update_layout(
         title_text=f"Phase correction Qubit {qubits[0]}",
         showlegend=True,
-        xaxis1_title="theta [rad] + virtual phase[rad]",
-        xaxis2_title="theta [rad] + virtual phase [rad]",
-        yaxis_title="Signal [a.u.]",
+        xaxis1_title="Virtual phase[rad]",
+        xaxis2_title="Virtual phase [rad]",
+        yaxis_title="Probability of State 0",
```

Comment:
Sounds better to me, feel free to ignore
```suggestion
        yaxis_title="State 0 probability ",
```